account-integrations/safe/test/WebAuthn.t.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Will the `WebAuthn.sol` file eventually live in the `primitives` directory?  I feel like this could be a good fit for that

Reply:
Yeah I think so good shout. Issue to do that - https://github.com/getwax/wax/issues/55